src/plugins/auto_batch/auto_batch.cpp

```diff
@@ -109,7 +109,17 @@ void AutoBatchInferRequest::ShareBlobsWithBatchRequest() {
                 _batchId,
                 _batchSize);
             break;
-
+        case InferenceEngine::Precision::U64:
+                res = create_shared_blob_on_top_of_batched_blob<InferenceEngine::Precision::U64>(
+                        _myBatchedRequestWrapper._inferRequestBatched->GetBlob(it.first),
+                        _batchId,
+                        _batchSize);
+        case InferenceEngine::Precision::I64:
+            res = create_shared_blob_on_top_of_batched_blob<InferenceEngine::Precision::I64>(
+                    _myBatchedRequestWrapper._inferRequestBatched->GetBlob(it.first),
+                    _batchId,
+                    _batchSize);
+            break;
```

Comment:
what tests are fixed?

Reply:
Should we extend these tests for auto plugin? https://github.com/openvinotoolkit/openvino/blob/master/src/tests/functional/plugin/shared/include/behavior/ov_infer_request/io_tensor.hpp#L47

Reply:
these tests were not instantiated even for the GPU- so I've added them, notice that u16/i16 is fixed by the https://github.com/openvinotoolkit/openvino/pull/10180/ so it is commented in the tests